_data-prepper/getting-started.md

```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
 b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
 ```
 The remainder of this page provides examples for running Data Prepper from the Docker image. If you
-built from source, refer to the [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md) for more information.
+built it from source, refer to the [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md) for more information.
 
-However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
-image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+However you configure your pipeline, you'll run Data Prepper the same way. You run the Docker
+image and modify both the `pipelines.yaml` and `data-prepper-config.yaml` files.
 
 For Data Prepper 2.0 or later, use this command:
 
 ```
 docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
 ```
 
-For Data Prepper before version 2.0, use this command:
+For Data Prepper 2.0 and earlier, use this command:
```

Comment:
This changes the meaning of the sentence.

Old: Version < 2.0
New: Version ≤ 2.0

Recommend changing to "For Data Prepper versions earlier than 2.0..." or something similar.

Reply:
`opensearchproject/data-prepper:1.x` won't pull 2.0 so I think we should definitely fix that.